test/spec/functional/viewActivation.js

```diff
@@ -0,0 +1,242 @@
+var testSrcPath = '../../source',
+    spyOnBackbone = require('./backboneSpy');
+
+describe('A View being deactivated and activated', function() {
+
+  var env, _, $, View, ClickView;
+
+  beforeEach(function(done) {
```

Comment:
I'm weary of making things too hidden or complex. I think sometimes having to be more explicit is more clear.